tests/unit/submissionworker/test_worker.py

```diff
@@ -0,0 +1,61 @@
+import boto3
```

Comment:
Can you please rename the file path to be `tests/unit/worker/test_submission_worker.py`